### PR TITLE
Be s3 eric

### PR DESCRIPTION
**Pull request by Ethorf**
Check out the s3 function in /utils!

### CODE REVIEW

**Review by surhud004 — COMMENTED** [review]
So far so good!

**Review by ChianHuei — APPROVED** [review]
When you integrate this feature with FE, you may modify client/src/form/AddNewListBar.js. This is the form where a user should upload an image file as a cover image.  Now it's just a textfield. Besides, you may check client/src/state_management/actionCreators/shoppingListsActs.js. The `addNewList` function here is to deal with this POST request.

**Comment by Ethorf** [comment]
Fantastic I will absolutely do that! Merging to dev

File: server/utils/fileUpload.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		cb(new Error('Invalid Mime Type, only JPEG and PNG'), false);
+	}
+};
+
+const upload = multer({
+	fileFilter,
+	storage: multerS3({
+		s3: s3,
+		bucket: 'team-watermelon-bigdeal-images',
+		//this is breaking it now
```
Comment (sina-jamshidi) [inline server/utils/fileUpload.js:27]: wait what does this mean lol

Comment (Ethorf) [reply server/utils/fileUpload.js:27]: Lol whoops! Leftover comment from when it was previously breaking, will delete.

**Pull request merged**